### PR TITLE
Code of conduct, many other updates.

### PR DESCRIPTION
**Pull request by bennyvasquez**
* Introduced a new, single code of conduct to address both digital spaces and physical spaces.
* Removed language that was more internal policy for employees than external CoC language.
* Removed language around helping community members in crisis that opened Progress up to legal concerns.
* Clarified who the CoC applies to and when
* removed content from the physical spaces CoC
* cleaned up links and references to various parts of the CoC

### CODE REVIEW

File: communication/community-meetings.md [hunk 1]
```diff
@@ -7,12 +7,11 @@ Meetings are scheduled for 45 minutes.
 ## Requirements
 
 - Meetings are open to anyone who is interested.
-- Each meeting must have a designated leader.
-- Each meeting must have an agenda posted in #community-meetings in Community Slack. Before posting, the agenda should be created in the community-slack-meetings GitHub repo at <https://github.com/chef/chef-community-slack-meetings>. Agendas should have draft time slots for each topic.
-- Putting together the meeting agenda is the responsibility of the designated leader.
+- Each meeting is lead by a member of the Developer Relations team in the Chef business unite at Progress.
```
Comment (damacus) [inline communication/community-meetings.md:10]:
```suggestion
- Each meeting is lead by a member of the Developer Relations team in the Chef business unit at Progress.
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,255 +1,139 @@
...
-- A **Community Member** is anyone who participates with the community
-  whether in-person or via online channels. Community members are responsible
-  for following the community guidelines, suggesting updates to the guidelines
-  when warranted, and helping enforce community guidelines
-
-| Role               | Name          | Contact Info     |
-| ------------------ | ------------- | ---------------- |
-| Decider            | benny Vasquez | bvasquez@chef.io |
-| Community Advocate | TBD           |                  |
+Participants asked to stop any harassing behavior are expected to comply immediately. Further unacceptable behavior will result in escalating sanctions.
```
Comment (Xorima) [inline CODE_OF_CONDUCT.md:48]:
```suggestion
Participants asked to stop any harassing behaviour are expected to comply immediately. Further unacceptable behavior will result in escalating sanctions.
```

Comment (ramereth) [reply CODE_OF_CONDUCT.md:48]: :roll_eyes: :gb:

**Comment by damacus** [comment]
LGTM 🚀

**Review by Xorima — APPROVED** [review]

**Pull request merged**